README.ru.md

```diff
@@ -290,9 +290,13 @@ hosts:
 
 # параметры фейловера
 failover:
-  mode: stateful                      # (опционально) вариант работы фейловера (stateful, eventual, disabled)
-  state_provider: stateboard          # (опционально) провайдер предоставляющий фейловер (stateboard, stateful)
-  stateboard_params:                  # (опционально) параметры для провайдера
+  mode: stateful              # (опционально) вариант работы фейловера (stateful, eventual, disabled)
+  state_provider: stateboard  # (опционально) провайдер предоставляющий фейловер (stateboard, stateful)
+  failover_timeout: 60        # (опционально) время (в секундах), определяет время перевода инстанса в статус `dead`
+  fencing_enabled: true       # (опционально) включение режима fencing
```

Comment:
There is much less information available comparing to the English doc you wrote. Let's close the gap, especially cuz we prioritize Russian docs a little more.

Reply:
You're right, it's done